Gemfile

```diff
@@ -1,9 +1,13 @@
 source 'https://rubygems.org'
 gemspec
 
+
 gem 'jruby-openssl', :platforms => :jruby
 
 group :test, :remote_test do
   # gateway-specific dependencies, keeping these gems out of the gemspec
-  gem 'braintree', '>= 2.50.0'
+gem 'braintree', '>= 2.50.0'
+gem 'builder', '~> 3.2.2'
```

Comment:
Why use `builder`? ActiveMerchant seems to favour Nokogiri for all XML generation/parsing